src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (invertedRobot.hasOwnProperty(robot[key])) {
```

Comment:
The use of 'hasOwnProperty' is correct for checking if a property exists on an object. However, since ECMAScript 5, it's recommended to use 'Object.prototype.hasOwnProperty.call(invertedRobot, robot[key])' to avoid issues when the object has a property with the name 'hasOwnProperty'.